components/HomePage/components.tsx

```diff
@@ -0,0 +1,462 @@
+import styled from "@emotion/styled";
+import { constants } from "../../utils";
+import { DiagonalArrowIcon } from "../Icons";
+import { Link, PrimaryLink, SecondaryLink } from "../Link";
+
+const { QUERIES, COLORS } = constants;
+
+export const Grid = styled.div`
+  display: flex;
+  flex-direction: column;
+`
+
+export const GridRow = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+`
+
+export const GridColumn = styled.div`
+  width: 50%;
+  padding: 0 10px;
+`
+
+export const Container = styled.section`
+  background-color: var(--gray-700);
+`
+
+export const Content = styled.div`
+  margin: auto;
+  padding: 50px 15px 45px;
+  max-width: 1200px;
+
+  @media ${QUERIES.tabletAndUp} {
+    padding: 115px 40px 75px;
+  }
+`
+
+export const Title = styled.h1`
+  display: block;
+  max-width: 320px;
+  margin: 0 auto 50px;
+  font-weight: 700;
+  font-size: clamp(2.5rem, 5.7vw + 0.2rem, 5rem);
+  line-height: 1;
+  text-align: center;
+  color: var(--white);
+
+  & > span {
+    color: var(--primary);
+  }
+
+  @media ${QUERIES.tabletAndUp} {
+    max-width: unset;
+    margin: 0 auto 80px;
+    line-height: 1.05;
+  }
+`;
+
+export const HomeSectionContainer = styled.section<{ grayBackground?: boolean }>`
+  background-color: ${({ grayBackground }) => grayBackground ? "var(--gray-300)" : "var(--white)"};
+`
+
+export const HomeSectionContent = styled.div`
+  padding: 25px 15px;
+  margin: auto;
+  max-width: 1200px;
+  
+  @media ${QUERIES.tabletAndUp} {
+    padding: 40px 20px 50px;
+  }
+
+  @media ${QUERIES.laptopAndUp} {
+    padding: 40px 40px 50px;
+  }
+`
+
+export const HomeSectionHeadline = styled.h2`
+  font-style: normal;
+  font-size: 16px;
+  line-height: 24px;
+  font-weight: 600;
+  text-transform: uppercase;
+  color: var(--primary);
+`
+
+export const OODescription = styled.p`
+  margin: 15px 0 30px;
+  font-style: normal;
+  font-size: 20px;
```

Comment:
Nit: let's put this in rem (but define it in px) by doing font-size: ${ 20 / 16}rem

Reply:
I replaced all texts dimensions from px to rem